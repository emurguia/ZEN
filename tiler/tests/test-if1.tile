// This tests that the various if control flow works as expected

init {
	bool x;
	bool y;

	x = true;
	y = !x;

	tile(3,3);

	if(x){
		sprint("Twinkle, twinkle");
	}

	if(y){
		sprint("Lalala, lalala, Elmo's World");
	}

	if(!y){
		sprint("Little star");
	}

	if(3 > 2){
		sprint("How I wonder what you are");
	}

	if((1+1) == 3){
		sprint("Bow chika wow wow");
	}

	close();

}