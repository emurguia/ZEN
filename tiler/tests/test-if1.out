Twinkle, twinkle
Little star
How I wonder what you are
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.